### PR TITLE
update readme and sphinx doc

### PR DESCRIPTION
**Pull request by benjicanal**

### CODE REVIEW

File: pyScienceMode/acks.py [hunk 1]
```diff
@@ -22,6 +34,15 @@ def motomed_error_values(error_code: int):
 
 def rehastim_error(error_code: int) -> str:
     """
+    Handles the errors of the Rehastim
```
Comment (Kev1CO) [inline pyScienceMode/acks.py:37]: Rehastim errors

File: pyScienceMode/acks.py [hunk 2]
```diff
@@ -34,6 +55,13 @@ def rehastim_error(error_code: int) -> str:
 
 def stimulation_error(error_code: int) -> str:
     """
+    Parameters
+    ----------
+    error_code: int
+        Error code of the Rehastim
```
Comment (Kev1CO) [inline pyScienceMode/acks.py:61]: Rehastim code error

File: pyScienceMode/utils.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import crccheck
 from .enums import ErrorCode, Rehastim2Commands
 
+# This code provides utility functions for working with the Rehastim device, including packet construction and data validation.
```
Comment (Kev1CO) [inline pyScienceMode/utils.py:4]: in """
"""
instead of #

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import sys
+
+sys.path.insert(0, os.path.abspath("../"))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "pyScienceMode"
+copyright = "2023, Ceglia, Co, Faresin"
+author = "Ceglia, Co, Faresin"
```
[the remark this answers is not in the document]
Comment (Kev1CO) [reply docs/conf.py:23]: order ?

**Review by Kev1CO — CHANGES_REQUESTED** [review]

File: docs/about.rst [hunk 5]
```diff
@@ -0,0 +1,10 @@
+======
+About
+======
+
+pyScienceMode is a python module to control the Rehastim2 and RehastimP24 devices from Hasomed.
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply docs/about.rst:5]: Done.

File: docs/index.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Welcome to pyScienceMode's documentation!
+==========================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   about
+   install
+   example
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply docs/index.rst:15]: Renamed in 'examples'

File: pyScienceMode/acks.py [hunk 7]
```diff
@@ -4,6 +4,18 @@
 
 
 def motomed_error_values(error_code: int):
+    """
+    Handles the errors of the Motomed
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply pyScienceMode/acks.py:8]: Done.

File: pyScienceMode/acks.py [hunk 8]
```diff
@@ -4,6 +4,18 @@
...
+
+    Parameters
+    ----------
+    error_code: int
+        Error code of the Motomed
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply pyScienceMode/acks.py:13]: Done.

File: pyScienceMode/acks.py (hunk 1 above)
Comment (benjicanal) [reply pyScienceMode/acks.py:37]: Done.

File: pyScienceMode/acks.py [hunk 9]
```diff
@@ -22,6 +34,15 @@ def motomed_error_values(error_code: int):
 
 def rehastim_error(error_code: int) -> str:
     """
+    Handles the errors of the Rehastim
+
+    Parameters
+    ----------
+    error_code: int
+        Error code of the Rehastim
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply pyScienceMode/acks.py:42]: Done.

File: pyScienceMode/acks.py (hunk 2 above)
Comment (benjicanal) [reply pyScienceMode/acks.py:61]: Done.

File: pyScienceMode/utils.py (hunk 3 above)
Comment (benjicanal) [reply pyScienceMode/utils.py:4]: Done.

File: docs/main_differences.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+| Communication speed                                                  | [20, 100] **ms**                                                                                       | [5,15] **ms**                                                                                    | 
+| Interpulse interval                                                  | 8 ms per stimulation module                                                                            | 5ms (can be modified)                                                                            | 
+| Serial port                                                          | USB with galvanic isolation                                                                            | USB Type-C                                                                                       |
+| Behavior when several channels are activated with too high frequency | The stimulator will stimulate at the maximum frequency to satisfy the offset between several channels. | Stimulation will not be active and the Rehastimp24 LED will be green.                            |
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply docs/main_differences.md]: Put a .png instead

File: docs/contributing.rst [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+`pyScienceMode` tries to follow as much as possible the PEP recommendations (https://www.python.org/dev/peps/).
+Unless you have good reasons to disregard them, your pull-request is required to follow these recommendations.
+
+All variable names that could be plural should be written as such.
+
+Black is used to enforce the code spacing.
+`pyScienceMode` is linted with the 120-character max per line's option.
+This means that your pull-request tests on GitHub will appear to fail if black fails.
+The easiest way to make sure black is happy is to locally run this command:
```
[the remark this answers is not in the document]
Comment (benjicanal) [reply docs/contributing.rst:40]: Done.

**Pull request merged**